frontend/www/js/omegaup/components/contest/Report.vue

```diff
@@ -11,11 +11,11 @@
       >
         <h1>{{ T.username }}: {{ contestantData.username }}</h1>
         <h3>
-          {{ T.wordsTotal }}:
-          <span v-if="hasTotalAndPointsProperties(contestantData)">
-            {{ contestantData.total.points }}
-          </span>
-          <span v-else>0</span>
+          {{
+            ui.formatString(T.contestReportTotalPoints, {
+              total_points: getTotalPoints(contestantData.total.points),
```

Comment:
```suggestion
              total_points: contestantData.total.points ?? 0,
```

acá no vale la pena declarar una función para esto.

Reply:
Ahh, el problema aquí es que al compilador no le gusta el uso de optional chaining en los templates de vue:
```ts
   ERROR in ./frontend/www/js/omegaup/components/contest/Report.vue?vue&type=template&id=7659da26&scoped=true& (./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/vue-loader/lib/index.js??vue-loader-options!./frontend/www/js/omegaup/components/contest/Report.vue?vue&type=template&id=7659da26&scoped=true&)
   Module build failed (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
   SyntaxError: Unexpected token (1:517)
       at Parser.pp$4.raise (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2757:13)
       at Parser.pp.unexpected (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:647:8)
       at Parser.pp$3.parseExprAtom (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2196:10)
       at Parser.<anonymous> (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:6003:24)
       at Parser.parseExprAtom (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:6129:31)
       at Parser.pp$3.parseExprSubscripts (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2047:19)
       at Parser.pp$3.parseMaybeUnary (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2024:17)
       at Parser.pp$3.parseExprOps (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:1966:19)
       at Parser.pp$3.parseMaybeConditional (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:1949:19)
       at Parser.pp$3.parseMaybeAssign (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:1925:19)
       at Parser.pp$3.parseMaybeConditional (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:1954:28)
       at Parser.pp$3.parseMaybeAssign (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:1925:19)
       at Parser.pp$3.parsePropertyValue (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2443:87)
       at Parser.pp$3.parseProperty (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2434:8)
       at Parser.pp$3.parseObj (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2380:23)
       at Parser.pp$3.parseExprAtom (/opt/omegaup/node_modules/vue-template-es2015-compiler/buble.js:2179:17)
 
   frontend (webpack 5.8.0) compiled with 1 error in 61660 ms
```

Reply:
GAH por eso lo introdujimos para empezar. tenemos que migrar a Vue 3 para que esto funcione: https://vue-next-template-explorer.netlify.app/#%7B%22src%22%3A%22%3Cdiv%20id%3D%5C%22app%5C%22%20v-if%3D%5C%22obj%3F.a%5C%22%3E%7B%7B%20msg%20%7D%7D%3C%2Fdiv%3E%22%2C%22options%22%3A%7B%22mode%22%3A%22module%22%2C%22filename%22%3A%22Foo.vue%22%2C%22prefixIdentifiers%22%3Afalse%2C%22hoistStatic%22%3Afalse%2C%22cacheHandlers%22%3Afalse%2C%22scopeId%22%3Anull%2C%22inline%22%3Afalse%2C%22ssrCssVars%22%3A%22%7B%20color%20%7D%22%2C%22compatConfig%22%3A%7B%22MODE%22%3A3%7D%2C%22whitespace%22%3A%22condense%22%2C%22bindingMetadata%22%3A%7B%22TestComponent%22%3A%22setup-const%22%2C%22setupRef%22%3A%22setup-ref%22%2C%22setupConst%22%3A%22setup-const%22%2C%22setupLet%22%3A%22setup-let%22%2C%22setupMaybeRef%22%3A%22setup-maybe-ref%22%2C%22setupProp%22%3A%22props%22%2C%22vMySetupDir%22%3A%22setup-const%22%7D%7D%7D

Reply:
tons que se quede así.

Reply:
Por ahí tengo un [PR ](https://github.com/omegaup/omegaup/pull/5426)algo olvidado para hacer la migración. Creo que es buen momento para retomarlo

Reply:
sí porfa :D